src/Nox.Reference.Data.World/Models/VatNumbers/ValidationStatus.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Nox.Reference;
 
-public enum VatValidationStatus
+public enum ValidationStatus
```

Comment:
May be leave as it was. ValidationStatus is more generic name for all validations

Reply:
It's a generic enum for validations in scope of this project